docs: Improve migration instructions (#49)
eduMFA/eduMFA: READ_BEFORE_UPDATE.md

# Update Notes

## Migrating from privacyIDEA 3.9.2 to eduMFA

> [!CAUTION]
>  * Make a backup of your existing installation! This backup should include the config files, the encryption keys and
     the database.
>  * Ensure that you use the latest supported privacyIDEA version 3.9.2 and then upgrade to eduMFA
> 

> > These migration steps are applicable for the versions 1.0.0 to 1.5.1

Reviewer: maybe we change this line until we release a version for which the instructions do NOT work
```suggestion
> These migration steps are applicable for all versions released up to now
```

> * Uninstall privacyIDEA and stop your nginx or `Apache2` or `nginx` service

Reviewer: ```suggestion
* Uninstall privacyIDEA and stop your `Apache2` or `nginx` service
```

* Move/Copy all configurations from `/etc/privacyidea/` to `/etc/edumfa/`
    * The config file needs to be renamed from `pi.cfg` to `edumfa.cfg`
    * Rename/Replace `PI_` in the configuration file with `EDUMFA_` (e.g. `PI_ENCFILE` changes to `EDUMFA_ENCFILE`)
    * Update the paths in the configuration file to `/etc/edumfa/`
    * Update the log file path to `/var/log/edumfa/edumfa.log`
* Install eduMFA (e.g. using Container, PIP or the `.deb` Package)

>     * When using the `.deb` package, make sure to not overwrite the existing configuration file

Reviewer: how? will the config be overwritten in any case?

Author: When having a config present, during installation you'll be asked if you like to keep the current config or use the maintainers version.

Reviewer: ```suggestion
    * When using the `.deb` package, make sure to not overwrite the existing configuration file (When asked by dpkg, answer N or O to keep your installed version)
```

Author: I think the current explanation should be mostly sufficient. Maybe we can add `during installation` and/or `when prompted`

Reviewer: OK
```suggestion
    * When using the `.deb` package, make sure not to overwrite the existing configuration file when prompted
```

>     * When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), all following steps will be performed automatically

Reviewer: I believe crontab or systemd services will not be updated by the .deb packages

Author: The packages adds a new cron, however systemd keeps unchanged. Maybe we should rather mention, what will be done automatically like: "When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will executed automatically. Additionally the web server configurtion and a cron will also be installed.

Reviewer: ```suggestion
    * When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will be executed automatically. Additionally the web server configuration and a cron entry will be installed.
```

* docs: start fixing incorrect camel case by @fritterhoff in https://github.com/eduMFA/eduMFA/pull/25
* docs: cleanup old refs and fix path names by @j-hoff in https://github.com/eduMFA/eduMFA/pull/28
* docs: use fu repo by @fritterhoff in https://github.com/eduMFA/eduMFA/pull/33
* docs: add docu for new sms provider "Http Message To Uid Provider" by @j-hoff

>   in https://github.com/eduMFA/eduMFA/pull/21

Reviewer 2: Is there a reason for the new line breaks? (Some formatting anywhere that requires a specific line length? 🤔 )

Author: This is the automated code format from Jetbrain IDEs

Reviewer 2: Would you mind to change your settings? There should be a setting for line breaks. Would prefer to keep the changes in existing changelogs as small as possible,

> ### Migrating from privacyIDEA 3.9.2 to eduMFA 1.0.0

Reviewer 2: Maybe it makes sense to shift this block to the top of the release notes and make it more generic? Migrating from privacyIDEA 3.9.2 to eduMFA
